lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -81,6 +81,12 @@
     - This is required if C(name) is not supplied.
     - If virtual machine does not exists, then this parameter is ignored.
     - Please note that a supplied UUID will be ignored on virtual machine creation, as VMware creates the UUID internally.
+  use_instance_uuid:
+    description:
+    - Use the VMWare instance UUID rather than the BIOS UUID.
+    default: 'no'
+    type: bool
+    version_added: 2.7
```

Comment:
This needs to become '2.8' now (as a string).
```suggestion
    version_added: '2.8'
```